src/frontend/components/design-system/molecules/drop-zone/drop-zone.tsx

```diff
@@ -24,11 +24,21 @@ const validateSize = (
   return +maxSize >= +size
 }
 
-const inKb = (size: string | number): string => {
+const inUnit = (size: string | number, unit: FileSizeUnit = 'kb'): string => {
   if (!size) { return '' }
-  return `${Math.round(+size / 1024)} KB`
+
+  let divider = 1024
+  if (unit === 'mb') {
+    divider = 1024 * 1024
+  }
+
+  console.log({ size, unit, divider })
+
+  return `${Math.round(+size / divider)} ${unit.toUpperCase()}`
 }
 
+type FileSizeUnit = 'kb' | 'mb';
```

Comment:
if you document it like that:

 * @memberof DropZone
 * @alias FileSizeUnit

it should be automatically put to the docs on the DropZone page